src/libraries/Microsoft.Extensions.DependencyInjection/src/ServiceLookup/CallSiteChain.cs

```diff
@@ -39,7 +39,7 @@ public void Add(Type serviceType, Type implementationType = null)
         private string CreateCircularDependencyExceptionMessage(Type type)
         {
             var messageBuilder = new StringBuilder();
-            messageBuilder.AppendFormat(SR.CircularDependencyException, TypeNameHelper.GetTypeDisplayName(type));
+            messageBuilder.Append(SR.Format(SR.CircularDependencyException, TypeNameHelper.GetTypeDisplayName(type)));
             messageBuilder.AppendLine();
```

Comment:
Shouldn't this instead have been:
```C#
messageBuilder.AppendLine(SR.Format(SR.CircularDependencyException, TypeNameHelper.GetTypeDisplayName(type)));
```
?  I expect it was only two separate calls previously because there isn't an AppendFormatLine method.

Reply:
It could be.